NSS-Day-Cohort-49/tabloid-full-stack-spicy-sweet-chilis
# My post sx

#### Changes Made
1. Added file `PostDetail.js`, `CurrentUserPostList.js` and `CurrentUserPost.js` to `Post` directory.
1. Modified file `PostRepository`, `IPostRepository`, and `PostController` to include `GetPostByCurrentUser` and `GetPostById` functionality.
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout myPost-sx
    ```
2. Open a new Terminal tab (?T) and navigate to the server directory.
3. Test app functionality.
    > My Post affordance should give you a list of posts you have posted...
    > While viewing all post, when clicking the title of each post, it'll take you to the post details page.
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
